packages/tradershub/src/features/cfd/modals/VerificationFailedModal/VerificationFailedModal.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Modal } from '../../../../components';
+import { VerificationFailed } from '../../screens';
+
+const VerificationFailedModal = () => {
+    return (
+        <Modal>
+            <Modal.Content>
+                <VerificationFailed />
+            </Modal.Content>
+        </Modal>
+    );
+};
```

Comment:
small nitpick can we remove the curly braces and the return aswell sorry just realize this 😭 🙏🏼